invenio_communities/communities/services/config.py

```diff
@@ -38,6 +40,15 @@
 )
 
 
+def _is_action_available(community, context):
```

Comment:
Very minor:
```suggestion
def can_perform_action(community, context):
```
Should this be moved to a `permissions.py`?